docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #79.                T[start : start + m],
</pre><p>Shouldn't this be <code>T[query_idx, query_idx + m]</code> ?</p><p> </p><p>Otherwise, the foreward code looks very neat and tidy. Though, I would put the backward function first and then the foreward function (right now foreward is first)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>You are right! I will fix Line #79. Thanks for catching it!</p><p> </p><blockquote>Though, I would put the backward function first and then the foreward function (right now foreward is first)</blockquote><p>One thing that I would like to point out here is that <code>_backward_process</code> is a mandatory step and <code>_forward_process </code> is an optional step that can speed up the process by pruning. So, I was wondering if we should keep the optional function at top and the two other functions, i.e. the main function and the <code>_backward_process </code> , next to each other. </p><p> </p><p>What do you think? </p><p> </p>

Reply:
<p>I get your point and maybe I'm overthinking things. As I'm reading from top to bottom, seeing <code>_foreward_process</code> first makes me think that it's more important than <code>_backward_processing</code> and I don't think that's the case, right?  </p>

Reply:
Correct. Since we are dealing with left neighbors (in classic `DAMP`), `_backward_process` is more important. You can even remove the `_forward_process` as the function is just for speeding up the computation. 

I will put `_backward_process` at the top. Later if you change your mind, I will change it back to current format.